tsl/test/sql/dist_am-12.sql

```diff
@@ -0,0 +1,58 @@
+-- This file and its contents are licensed under the Timescale License.
```

Comment:
The -12 suffix is not needed on this file. (In fact, we probably shouldn't have it since it will run also on, e.g., PG 13 once released). It's enough to exclude this test from running on pre-PG12.